Maple2.File.Parser/TableParser.cs

```diff
@@ -336,7 +355,7 @@ public IEnumerable<UnitedWeeklyReward> ParseUnitedWeeklyReward() {
         Debug.Assert(data != null);
 
         foreach (Fish fish in data.fish) {
-            yield return (fish.id, fishNames.GetValueOrDefault(fish.id), fish);
+            yield return (fish.id, fishNames.GetValueOrDefault(fish.id) ?? string.Empty, fish);
```

Comment:
Is this needed? Also, shouldn't we specify the default within the GetValueOrDefault function?

Reply:
it returns `string?`